web/README.md

```diff
@@ -1,28 +1,140 @@
+
+When relevant, Netdata prefers to organize charts by family. When you visit the **Disks** section, you will see your disk drives organized into families, and each family will have one or more charts: `disk`, `disk_ops`, `disk_backlog`, `disk_util`, `disk_await`, `disk_avgsz`, `disk_svctm`, `disk_mops`, and `disk_iotime`.
+
+In the screenshot below, the disk family `sdb` shows a few gauges, followed by a few of the associated charts:
+
+![Screenshot of a disk drive family and associated charts in the Netdata dashboard](https://user-images.githubusercontent.com/1153921/62721362-e34f5f80-b9c0-11e9-8d2e-9a3bec48e920.png)
+
+Netdata also creates separate submenu entries for each family in the right navigation page so you can easily navigate to the instance you're interested in. Here, Netdata has made several submenus under the **Disk** menu.
+
+![Screenshot of the disks menu and submenus](https://user-images.githubusercontent.com/1153921/62721531-3cb78e80-b9c1-11e9-89c2-fdd736aec7d4.png)
+
+### Contexts
+
+A **context** is a way of grouping charts by the types of metrics collected and dimensions displayed. Different charts with the same context will show the same dimensions, but for different instances (families) of hardware/software resources.
+
+For example, the **Disks** section will often use many contexts (`disk.io`, `disk.ops`, `disk.backlog`, `disk.util`, and so on). Netdata then creates an individual chart for each context, and groups them by family.
+
+Netdata names charts according to their context according to the following structure: `[context].[family]`. A chart with the `disk.util` context, in the `sdb` family, gets the name `disk_util.sdb`. Netdata shows that name in the top-left corner of a chart.
+
+Given the four example contexts, and two families of `sdb` and `sdd`, Netdata will create the following charts and their names:
+
+Context | `sdb` family | `sdd` family
+--- | --- | ---
+`disk.io` | `disk_io.sdb` | `disk_io.sdd`
+`disk.ops` | `disk_ops.sdb` | `disk_ops.sdd`
+`disk.backlog` | `disk_backlog.sdb` | `disk_backlog.sdd`
+`disk.util` | `disk_util.sdb` | `disk_util.sdd`
+
+And here's what two of those charts in the `disk.io` context look like under `sdb` and `sdd` families:
+
+![context_01](https://user-images.githubusercontent.com/1153921/62728232-177e4c80-b9d0-11e9-9e29-2a6c59d4d873.png)
+![context_02](https://user-images.githubusercontent.com/1153921/62728234-1b11d380-b9d0-11e9-8904-07befd8ac592.png)
+
+As you can see in the screenshot, you can view the context of a chart if you hover over the date above the list of dimensions. A tooltip will appear that shows you two pieces of information: the collector that produces the chart, and the chart's context.
+
+Netdata also uses contexts for alarm templates. You can create an alarm for the `net.packets` context to receive alerts for any chart with that context, no matter which family it's attached to.
+
+### Positive and negative values on charts
```

Comment:
Header should be one level higher, currently appears under Charts, contexts, families